hack/common

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+alias oc=${OC:-oc}
+
+repo_dir="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/.."
+
+IMAGE_TAG=${IMAGE_TAG:-"quay.io/openshift/origin-elasticsearch-proxy:latest"}
+
+function login_to_registry() {
```

Comment:
I  have always questioned the need for any of this complicated logic.  This is a dev operation.  Is there any reason we simply can not assume the user running the target can also push to the registry in order to simplify the logic.  Can we just not fail with a simple message that "Are you certain you can log into XXXXX". My preference would be to login simply with something like`oc login -u $(oc whoami) -p $(oc whoami -t)`

Reply:
what if i'm running locally as `system:admin`, i'll be required to log in as `kubeadmin` then when i want to push an image? i currently like using the `PUSH_USER` and `PUSH_PASSWORD` mechanic

Reply:
This seems rare IMO and I have seen problems myself when developing locally where the context is not being reset correctly